packages/strapi-admin/admin/src/containers/PluginDispatcher/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ *
+ * PluginDispatcher
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import Helmet from 'react-helmet';
+import BlockerComponent from 'components/BlockerComponent';
+import ErrorBoundary from 'components/ErrorBoundary';
+
+function PluginDispatcher(props) {
+  const {
+    global: { plugins },
+    match: { params: { pluginId } },
+  } = props;
+  
+  const pluginToRender = get(plugins, pluginId, null);
+
+  if (!pluginToRender) {
+    return null;
+  }
+  
+  const {
+    mainComponent,
+    name,
+    preventComponentRendering,
+  } = pluginToRender;
+  const blockerComponentProps = pluginToRender.blockerComponentProps;
+  let Compo = preventComponentRendering ? BlockerComponent : mainComponent;
```

Comment:
You could find a better naming for this variable

Reply:
https://github.com/strapi/strapi/blob/f5ab26fb466741abc9d1d4e60c2788840001bd12/packages/strapi-admin/admin/src/containers/PluginDispatcher/index.js#L32